### DISCUSSION TITLE
DLしたフォントを画像生成で利用できるのか、調査する

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#177] [90: 知識・知見の共有]
Webフォントだと画像生成で使用できるが、DLしたフォントでも可能なのか？

## 関連リンク
#173

**Comment by kazzyfrog** [comment]
fetchで、フォントのデータを取得するのは、対象のサイトがアクセスを許可していなかったので無理でしたが、
DLしたフォントなら、可能でした！

サーバーサイドでも、ブラウザ上でも、使えます！

検証用のプロジェクトでの例

サーバーでビルド時に生成：
![](https://kazzyfrog.github.io/nextjs-deploy/piyo/og.png)

ブラウザ上で生成：
「click」ボタンを押せば、入力した内容で生成されます！
https://kazzyfrog.github.io/nextjs-deploy/piyo

**Reply by kagomen** [reply]
調査ありがとうございます！！
すごい…！🙏✨
